lib/services.rb

```diff
@@ -22,7 +22,8 @@ def self.worldwide_api
 
   def self.content_store
     @content_store ||= GdsApi::ContentStore.new(
-      Plek.new.find('content-store')
+      Plek.new.find('content-store'),
+      timeout: 8
```

Comment:
It seems that this is likely to affect a lot of the frontend apps until they're all running in AWS. I've certainly seen it on government-frontend and finder-frontend as well as smart-answers.

I wonder if it's worth increasing the default timeout from 4 to 8 on [`GdsApi::ContentStore`](https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api/content_store.rb#L6) to allow more apps to take advantage of this? As long as we remember to decrease it again once the frontends have migrated.

Reply:
If we're going to do this then would it make more sense to use an environment variable and override it per app? If we make the change in `GdsApi::ContentStore` this would involve two releases (one to increase the timeout, another to revert the timeout), plus two releases for each of the apps consuming the gem. 

There could also be some inconsistencies if we have any apps already migrated to AWS using the extended timeout, which is not the change we're looking to make here.